rclcpp/include/rclcpp/graph_listener.hpp

```diff
@@ -160,6 +160,8 @@ class GraphListener : public std::enable_shared_from_this<GraphListener>
   void
   run_loop();
 
+  void init_wait_set();
```

Comment:
`RCLCPP_PUBLIC`?

Reply:
Good call, see 9f210213c2be2cb30ad7956da6bcbf5573db551f .  Windows CI failed for other reasons, but it probably would have failed here too eventually.